features/checkout/prevent_skipping_checkout_steps.feature

```diff
@@ -33,3 +36,29 @@ Feature: Prevent skipping checkout steps
         And I am at the checkout addressing step
         When I want to complete checkout
         Then I should be on the checkout addressing step
+
+    @ui
+    Scenario: Skipping addressing checkout step when order total is zero
+        Given I have product "PHP T-Shirt" in the cart
+        And I have product "Paganini T-Shirt" in the cart
+        And I am at the checkout addressing step
+        When I want to complete checkout
+        Then I should be on the checkout addressing step
+
+    @ui
+    Scenario: Skipping shipping checkout step when order total is zero
```

Comment:
Scenario name is confusing. Should be smth like: "Not being able to skip the checkout shipping selection step when order total is zero".